text/0000-core-subteam.md

```diff
@@ -0,0 +1,106 @@
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+The Core subteam is Responsible for the design, architecture, performance,
+stability, and security of the core Grid platform.
+
+The Core subteam exists for the purpose of technical review and decision
+making, in particular those which are in the form of an RFC.
+
+Non-technical decisions should be left to the Root subteam or other subteam as
+appropriate.  For example, as new components are added to Grid, the Core
+subteam may act in a capacity of technical review of the potentially incoming
+components. However, the decision on whether to add the component may be left
+with the Root subteam as a wider project decision. Thus it is possible that
+RFCs may require approval from both the Root and Core subteams, if that is
+deemed appropriate.
+
+As with the other subteams detailed in the governance RFC, the specific
+decisions the Core subteam will be tasked with include:
+
+- Shepherding applicable RFCs
+- Coming to consensus on whether to accept or reject RFCs
+- Setting policy on what sort of changes require an RFC
+- Coming to consensus on new members for the subteam
+- Reviewing PRs
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+In order to achieve the long-term architectural consistency, membership in the
+Core team should consist of individuals who have a fairly long history with the
+project. For the subteam to be able to properly evaluate RFCs or decisions with
+the proper context, the subteam membership should consist of members who have
+a working knowledge of the existing code and/or overall existing architecture.
+
+Therefore, the initial membership of the Core subteam is made up of Root team
+members who are Grid maintainers and/or are have a history of providing input
+into Grid technical issues:
+
+- @vaporos, Shawn Amundson, team lead
+- @peterschwarz, Peter Schwarz
+- @jsmitchell, James Mitchell
+- @dplumb94, Darian Plumb
+- @desktophero, Jason Walker
+- @agunde406, Andrea Gunderson
+- @adeebahmed, Adeeb Ahmed
+
+While members of this subteam have decision making capability, it is critical
+to the health of the project that the wider community is involved in
+discussions and that the subteam members represent the wider community to
+the extent possible.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+None.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+No alternatives have been discussed.
+
+# Prior art
+[prior-art]: #prior-art
+
+This is based on the governance model laid out by
+[Grid RFC 0008](https://github.com/hyperledger/grid-rfcs/blob/master/text/0008-grid-governance.md),
+[Sawtooth RFC 0006](https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0006-sawtooth-governance.md),
+[Sawtooth RFC 0024](https://github.com/hyperledger/sawtooth-rfcs/blob/master/text/0024-core-subteam.md),
+and
+[Rust team](https://github.com/rust-lang/rfcs/blob/master/text/1683-docs-team.md).
```

Comment:
This RFC may be better/additional than/to the docs team:
https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md

Reply:
That particular one was referenced because it is an example of adding a subteam.